src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java

```diff
@@ -129,9 +133,9 @@ public void revokeAccessTokens(String id, String token) {
             LOGGER.warn(String.format("Unable to revoke access token on auth-server %s: %s",
                     authServerHome, e.getMessage()));
             throw e;
-        } catch(OsiamRequestException e) {
+        } catch (UnauthorizedException e) {
             LOGGER.warn("Error revoking all access tokens", e);
-            throw e;
+            throw new InvalidTokenException("The provided access token is not valid", e);
```

Comment:
The auth-server endpoint is not necessary here, but I agree not to log the stack trace.